BLSQ/iaso
# IA-2059:  Registry show multiple submissions

Org unit can have multiple submissions linked to it, it should be possible to see it in the registry.

Related JIRA tickets : IA-2059

Selected submission id should come from params  **OR** reference instance id **OR** first submission id of the possible submissions linked to the org unit **OR** undefined
 if undefined  a place holder should say no submission

## Self proofreading checklist


## Changes

- move instance part to a separate component
- fetch list of instances linked to the org unit
- select displaying list of submissions
- deep link of selected submission
- pre select submission if nothing in the params and a reference instance exists, or the first submission from the list
- fetch submission detail to get form descriptor
- display placeholder if no submission for the current org unit

## How to test

Make sure you have working registry with [this](https://github.com/BLSQ/iaso/tree/main/docs/pages/dev/how_to/create_registry_in_web_ui)

Find org unit with reference submission, one without but with other submissions and a last one without submission at all.
Visit those org units in the registry page.
Make sure the first one has the reference submission selected
Make sure the second one has the first submission of the list selected
The last one displays a placeholder

## Print screen / video


https://user-images.githubusercontent.com/12494624/234525301-2c087162-0289-4459-9693-009e4121293b.mov

<img width="1656" alt="Screenshot 2023-04-26 at 10 51 01" src="https://user-images.githubusercontent.com/12494624/234525349-f21f59a5-b87f-4d0b-9af2-46fee0b45975.png">

## Notes

JIRA ticket is talking about multiple reference submission, i'm sure if this is doable.
